Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/day/sheets/location/ContactDiaryLocationBottomSheetDialogViewModel.kt

```diff
@@ -36,6 +38,25 @@ class ContactDiaryLocationBottomSheetDialogViewModel @AssistedInject constructor
         shouldClose.postValue(null)
     }
 
+    fun updateLocation(location: ContactDiaryLocationEntity) = launch {
+        contactDiaryRepository.updateLocation(
+            ContactDiaryLocationEntity(
```

Comment:
`de.rki.coronawarnapp.contactdiary.model.DefaultContactDiaryLocation` and `de.rki.coronawarnapp.contactdiary.model.DefaultContactDiaryPerson` have been re added on the target branch after they got lost, could we use those instead of the entity directly?

Reply:
The repository expects an entity

Reply:
no? https://github.com/corona-warn-app/cwa-app-android/blob/ca66dbf74ce8209fe6ee91590270d26d30ae3bcf/Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/storage/repo/ContactDiaryRepository.kt#L15
it expects something implemeting `ContactDiaryLocation` which `DefaultContactDiaryLocation` does
https://github.com/corona-warn-app/cwa-app-android/blob/ca66dbf74ce8209fe6ee91590270d26d30ae3bcf/Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/model/DefaultContactDiaryLocation.kt#L3-L7

Reply:
I refactored everything when this was removed

Reply:
changed it